src/createVersionFile.proj

```diff
@@ -0,0 +1,9 @@
+<Project DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+
+  <Import Project="dir.props" />
```

Comment:
There is no such file in src that you are trying to import. did you mean "../dir.props"?

the build failed here:

```
2017-05-25T00:03:24.0059617Z Build started 5/24/2017 5:03:24 PM.
2017-05-25T00:03:24.0215845Z Project "E:\A\_work\382\s\src\createVersionFile.proj" on node 1 (default targets).
2017-05-25T00:03:24.0215845Z E:\A\_work\382\s\src\createVersionFile.proj(3,3): error MSB4019: The imported project "E:\A\_work\382\s\src\dir.props" was not found. Confirm that the path in the <Import> declaration is correct, and that the file exists on disk.
2017-05-25T00:03:24.0215845Z Done Building Project "E:\A\_work\382\s\src\createVersionFile.proj" (default targets) -- FAILED.
2017-05-25T00:03:24.0215845Z 
```